docs/fast_data/connectors/debezium_cdc.md

```diff
@@ -525,6 +525,15 @@ debezium.transforms.Reroute.key.enforce.uniqueness=false
 
 </details>
 
+
+### Debezium Server Generic Template
+
+If none of the databases mentioned in the previous sections are being used, it is still possible to connect any custom database with the Debezium Server.
```

Comment:
I wouldn't say any custom databases but the ones supported by debezium that we don't provide as a service in the mkpl